components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/util/ClientCredentialAuthConfig.java

```diff
@@ -444,7 +457,7 @@ private Pair<RetryDecision, String> processSuccessfulResponse(CloseableHttpRespo
             if (LoggerUtils.isDiagnosticLogsEnabled()) {
                 DiagnosticLog.DiagnosticLogBuilder diagnosticLogBuilder = new
                         DiagnosticLog.DiagnosticLogBuilder(Constants.LogConstants.ADAPTIVE_AUTH_SERVICE,
-                        Constants.LogConstants.ActionIDs.RECEIVE_API_RESPONSE);
+                        getInvokeApiActionId(getRequest()));
```

Comment:
This is still related to retrieving an access token right. Is this action right ?

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/161/commits/3552a120da28ec992f1e3428119cf1498c8e2b33